internal/auth/jwt/jwtauth.go

```diff
@@ -124,6 +125,12 @@ func GetUserSubjectFromContext(ctx context.Context) string {
 	if !ok {
 		return ""
 	}
+	// TODO: wire this in to IdentityProvider interface.  Alternatively, have a different version
+	// for authzClient.Check that is IdentityProvider aware
+
+	if token.Issuer() == "https://token.actions.githubusercontent.com" {
+		return fmt.Sprintf("githubactions/%s", strings.ReplaceAll(token.Subject(), ":", "+"))
+	}
```

Comment:
Hahaha, this is gross.  We should probably put all of this mangling / un-mangling in `authzClient`, rather than putting some inside and some outside (which is the current state of play).